static/js/import_wizard/types.ts

```diff
@@ -46,6 +46,8 @@ export interface Column {
   header: string;
   // column index (leftmost column will be 0)
   columnIdx: number;
+  // unit of measure set for this column
```

Comment:
This feels a bit odd to me to have some mapped-thing specific field in Column, which is sort of a general structure. It is also odd that UNIT appears as a MappedThing when it is a constant, but not when it is specified in the header.

What do you think about having UNIT always appear as MappedThing, and then in MappingVal, when MappingType is COLUMN_HEADER, we use a new field:

```
export type UnitValue = string;

export interface MappingVal {
   ...
   unitValue?: { [columnIdx: number]: UnitValue };`
}
```

Reply:
Hm, I'm not sure how I feel about COLUMN_HEADER mappingType needing to read from a different field when the MappedThing is UNIT vs everything else. What about an additional MappingType like COLUMN_CONSTANT and then a field in MappingVal like `columnConstants?: { [columnIdx: number]: string}`?

Reply:
+1 I like that

Reply:
done